releasenotes/notes/deprecate-operator-httpfetch.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: release-notes/v2
+kind: feature
+area: installation
+releaseNotes:
+- |
+  **Deprecated** fetching charts from arbitrary URLs in `istio-operator`.
```

Comment:
Should we remove arbitrary here as well?

Reply:
sounds good